linux_os/guide/system/logging/log_rotation/timer_logrotate_enabled/rule.yml

```diff
@@ -1,5 +1,7 @@
 documentation_complete: true
 
+prodtype: sle12,sle15,rhel9
```

Comment:
```suggestion
prodtype: rhel9,sle12,sle15
```

Reply:
`ol9` can also be added to the `prodtype` list. It is the only OL release which currently supports the `logrotate.timer` service.

Reply:
Also worth mentioning: [Release notes](https://github.com/logrotate/logrotate/releases/tag/3.11.0) for logrotate's release 3.11.0 mentions:
> _Import systemd service and timer for logrotate from openSUSE._

So this feature might be available from this release, and not starting from 3.14.0 as suggested in some comment above.